nanshe_workflow/par.py

```diff
@@ -121,6 +122,55 @@ def get_client(profile):
     return(client)
 
 
+def map_ipyparallel(client, calculate_block, data, block_shape, block_halo):
+    block_indices, data_blocks, data_halo_blocks, result_halos_trim = split_blocks(
+        data.shape, block_shape, block_halo, index=True
```

Comment:
By setting `index`, we technically require `kenjutsu` 0.5.0+, but we forgot to add this. It comes later in PR ( https://github.com/nanshe-org/nanshe_workflow/pull/55 ), but is missing from the 2.0.7 release. That said, one can easily update if they want to use that build.